apps/Gruntfile.js

```diff
@@ -799,6 +822,12 @@ testsContext.keys().forEach(testsContext);
     'karma:integration'
   ]);
 
+  grunt.registerTask('scratchTest', [
+    'preconcat',
+    'concat',
+    'karma:scratch',
+  ]);
+
```

Comment:
Ahh, the different Blockly versions makes sense. Might be worth a comment somewhere (not sure on the best place) explaining this, so that the next person to create a new app doesn't think they should also be making a new test suite by default.